src/components/Star.js

```diff
@@ -0,0 +1,13 @@
+import * as PIXI from 'pixi.js'
```

Comment:
with pixi. v5 this is better to use named import (for the resulting bundle size):
import { Graphics } from 'pixi.js'